### PR TITLE
Convert mkuidefaults script to python3 and pep8 it

### PR DESCRIPTION
**Pull request by lbartoletti**
## Description
A tiny PR to "convert" mkuidefaults script to python3 and pep8 it

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

- [X] Commit messages are descriptive and explain the rationale for changes
- [ ] Commits which fix bugs include `fixes #11111` in the commit message next to the description
- [ ] Commits which add new features are tagged with `[FEATURE]` in the commit message
- [ ] Commits which change the UI or existing user workflows are tagged with `[needs-docs]` in the commit message and containt sufficient information in the commit message to be documented
- [X] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] New unit tests have been added for core changes
- [ ] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTE.md#contributing-to-qgis) before each commit

### CODE REVIEW

File: scripts/mkuidefaults.py [hunk 1]
```diff
@@ -39,7 +40,7 @@ def chunks(l, n):
 QCoreApplication.setApplicationName("QGIS3")
 
 if len(sys.argv) == 1:
-    print "Usage: python ./scripts/mkuidefaults.py \"location_to_ini\""
+    print("Usage: python ./scripts/mkuidefaults.py \"location_to_ini\"")
```
Comment (m-kuhn) [inline scripts/mkuidefaults.py]: Recommending `python ...` here and using `python3` as shebang on top looks inconsistent to me. Maybe better switch it to `Usage: ./scripts/mkuidefaults.py \"path_to_ini\"` would be better with the shebang?

**Comment by lbartoletti** [comment]
@m-kuhn is it ok for you? Thanks.

File: scripts/mkuidefaults.py [hunk 2]
```diff
@@ -49,33 +50,40 @@ def chunks(l, n):
 
 f = open("src/app/ui_defaults.h", "w")
```
Comment (m-kuhn) [inline scripts/mkuidefaults.py]: It would be really nice to have this in a `with open(...) as f:` block

**Comment by m-kuhn** [comment]
Looks good. It's an improvement already, there's just a suggestion.

**Comment by m-kuhn** [comment]
@lbartoletti do you plan to integrate the suggested change or should we merge as is?

**Comment by lbartoletti** [comment]
@m-kuhn done

**Comment by lbartoletti** [comment]
@m-kuhn ready to merge ? Thanks :)

**Comment by m-kuhn** [comment]
Thank you !
I was just waiting for travis (which last time had timed out, so I had to restart and forgot to check again after that ;) )

**Pull request merged**